dotnet/src/SemanticKernel.IntegrationTests/Planning/FunctionFlowPlannerTests.cs

```diff
@@ -4,121 +4,68 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.SemanticKernel;
 using Microsoft.SemanticKernel.Memory;
-using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Planning.Planners;
 using SemanticKernel.IntegrationTests.Fakes;
 using SemanticKernel.IntegrationTests.TestSettings;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace SemanticKernel.IntegrationTests.Planning;
 
-public sealed class PlanTests : IDisposable
+public sealed class FunctionFlowPlannerTests : IDisposable
 {
-    public PlanTests(ITestOutputHelper output)
+    public FunctionFlowPlannerTests(ITestOutputHelper output)
     {
-        this._logger = NullLogger.Instance; //new XunitLogger<object>(output);
+        this._logger = new XunitLogger<object>(output);
         this._testOutputHelper = new RedirectOutput(output);
 
         // Load configuration
         this._configuration = new ConfigurationBuilder()
             .AddJsonFile(path: "testsettings.json", optional: false, reloadOnChange: true)
             .AddJsonFile(path: "testsettings.development.json", optional: true, reloadOnChange: true)
             .AddEnvironmentVariables()
-            .AddUserSecrets<PlanTests>()
+            .AddUserSecrets<FunctionFlowPlannerTests>()
             .Build();
     }
 
     [Theory]
-    [InlineData("Write a poem or joke and send it in an e-mail to Kai.")]
-    public void CreatePlan(string prompt)
-    {
-        // Arrange
-
-        // Act
-        var plan = new Plan(prompt);
-
-        // Assert
-        Assert.Equal(prompt, plan.Description);
-        Assert.Equal(prompt, plan.Name);
-        Assert.Equal("Microsoft.SemanticKernel.Orchestration.Plan", plan.SkillName);
-        Assert.Empty(plan.Steps);
-    }
-
-    [Theory]
-    [InlineData("This is a story about a dog.", "kai@email.com")]
-    public async Task CanExecuteRunSimpleAsync(string inputToEmail, string expectedEmail)
+    [InlineData("Write a poem and send it in an e-mail to Kai.", "SendEmailAsync", "_GLOBAL_FUNCTIONS_")]
+    public async Task CreatePlanFunctionFlowAsync(string prompt, string expectedFunction, string expectedSkill)
```

Comment:
Make sure this passes on GPT3

Reply:
Green: https://github.com/lemillermicrosoft/semantic-kernel/actions/runs/4748412995